assets/js/app.js

```diff
@@ -0,0 +1,101 @@
+const navMenu = document.getElementById("fa");
+const getSideBar = document.querySelector(".nav-menu");
+const closeBtn = document.querySelector(".aside-close");
+const navLinks = document.querySelectorAll(".mobile-nav li");
+const moreWrapper  = document.querySelector(".more-wrapper")
+const hiddenSpeakers = document.getElementsByClassName("unhide")
+const toggleSideBar = false;
+const closeButton = false;
+const body = document.querySelector("body")
+navMenu.addEventListener("click", function () {
+  if (toggleSideBar === false) {
+    getSideBar.style.visibility = "visible";
+    body.style.overflow = 'hidden'
+  }
+});
```

Comment:
[OPTIONAL] **I highly recommend that you fix this issue with your hamburger menu**. When you click the button and select the 'About' option, It disables by default.
![hamburger_diables](https://user-images.githubusercontent.com/45668340/147255079-cc2caad9-a935-4d33-8a23-fa084f2d9cb4.gif)